src/liveEditor/__test__/withoutIframe.test.ts

```diff
@@ -26,9 +26,12 @@ describe("When outside the Visual editor, the Visual Editor", () => {
     afterAll(() => {
         mockedConsoleError.mockRestore();
     });
-    test("should have the start editing button", () => {
+    test("should have the start editing button", async () => {
         const config: IConfig = getDefaultConfig();
         new VisualEditor(config);
+
+        await sleep();
```

Comment:
Do we need this sleep?

Reply:
Yes, when the Live editor loads, we send an `init` event to the parent. If we don't receive anything from the parent, we assume we are outside the live editor, and thus, we show the button.

So, sleep is required to wait for the post-message call to send an error back.